store/cassandra/table.go

```diff
@@ -29,9 +30,14 @@ func GetTTLTables(ttls []uint32, windowFactor int, nameFormat string) TTLTables
 
 func GetTable(ttl uint32, windowFactor int, nameFormat string) Table {
 	/*
-	 * the purpose of this is to bucket metrics of similar TTLs.
-	 * we first calculate the largest power of 2 that's smaller than the TTL and then divide the result by
-	 * the window factor. for example with a window factor of 20 we want to group the metrics like this:
+	 * The purpose of this is to bucket metrics of similar TTLs in well-tuned tables.
+	 * We first calculate the preFactorWindow, which is the largest power of 2 that's
+	 * smaller or equal to the TTL in hours.
```

Comment:
are you sure about the `or equal`? just looking at the examples in the table below, that looks like it's just `smaller` and not `equal`:

```
4 <= hours < 8 |     metrics_4
```